packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.controller.js

```diff
@@ -169,4 +169,56 @@ export default class FlavorsListController {
   formatStorage(capacity) {
     return this.$filter('bytes')(capacity, 2, false, 'GB');
   }
+
+  /**
+   * Get the lowest hourly price and lowest monthly price for the flavor
+   * and return distinct prices for both hourly and monthly
+   * @param {*} flavor
+   * @returns Object with lowest hourly / monthly and all distinct hourly / monthly
+   */
+  static getLowestPriceFromFlavor(flavor) {
```

Comment:
**nitpick:** Since the function does not just return the lowest price. `getLowestPrice` does not make sense.